src/App.js

```diff
@@ -1,9 +1,8 @@
-
 import './style.css';
 
 function App() {
   return (
-    <div >
+    <div>
      <h1>Hello TODO APP</h1>
     </div>
```

Comment:
Well done Henry, The header looks great. I would only like to suggest you use the header semantic tag instead of the more generic div as it makes your app more accessible and optimal